consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/network/IbftMessageTransmitter.java

```diff
@@ -36,12 +39,12 @@
 public class IbftMessageTransmitter {
 
   private final MessageFactory messageFactory;
-  private final ValidatorMulticaster multicaster;
+  private final IbftGossip networkInterface;
```

Comment:
`networkInterface` is not a nice reference name. Perhaps `gossiper` or just `network`?

Reply:
done.